verify-certs/verify.go

```diff
@@ -0,0 +1,100 @@
+package main
```

Comment:
The validity of the passed certificate chain - that a certificate hasn't expired, that the certificates are actually valid, that it's signed by a trusted root CA. 

All `vhostcert` checks is that there's a chain of certs that is orderable in that each claims to be issued by the next.